doc/sphinx/addendum/nsatz.rst

```diff
@@ -44,7 +44,7 @@ files that declare the axioms used to define the real numbers, you can
 
    `radicalmax`
      bound when searching for r such that
-     :math:`c (P−Q) r = \sum_{i=1..s} S_i (P i − Q i)`.
+     :math:`c (P−Q)^r = \sum_{i=1..s} S_i (P_i − Q_i)`.
```

Comment:
What are the `S_i`?  I only see `s` above, the number of terms.

And what is `c` (also mentioned under "parameters")?  Is it a parameter or a result?